guides/doc-Configuring_Load_Balancer/docinfo.xml

```diff
@@ -1,7 +1,7 @@
 <title>Configuring Capsules with a Load Balancer</title>
 <productname>Red Hat Satellite</productname>
 <productnumber>6.14</productnumber>
-<subtitle>Distributing load between Capsule Servers</subtitle>
+<subtitle>Distribute load among Capsule Servers</subtitle>
```

Comment:
This is the only subtitle that uses 'Capsule Server' and not 'Capsule'. Is there a reason for that?

Reply:
I kept it because, there's "Capsules" instead of "Capsule Servers" in the main title and I wanted the subtitle to be more descriptive, but now I think it should perhaps stick with "Capsules" in the subtitle for the sake of unification :)